components/supervisor/pkg/supervisor/ssh.go

```diff
@@ -213,3 +214,37 @@ func writeSSHEnv(cfg *Config, envvars []string) error {
 
 	return nil
 }
+
+func configureSSHDefaultDir(cfg *Config) {
+	if cfg.RepoRoot == "" {
+		log.Error("cannot configure ssh default dir with empty repo root")
+		return
+	}
+	file, err := os.OpenFile("/home/gitpod/.bashrc", os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0o644)
+	if err != nil {
+		log.WithError(err).Error("cannot write .bashrc")
+	}
+	defer file.Close()
+	if _, err := file.WriteString(fmt.Sprintf("\nif [[ -n $SSH_CONNECTION ]]; then cd \"%s\"; fi\n", cfg.RepoRoot)); err != nil {
+		log.WithError(err).Error("write .bashrc failed")
+	}
+}
+
+func configureSSHMessageOfTheDay() {
+	msg := []byte(`Welcome to Gitpod - Always ready to code. You can use commands below to be more productive.
+
+	gp tasks list         List all your defined tasks in .gitpod.yml
+	gp tasks attach       Attach to a workspace task
+
+	gp ports list         Lists workspace ports and their states
+	gp timeout extend     Extend timeout of current workspace
+	gp top                Display usage of workspace resources (CPU and memory)
```

Comment:
Not sure it's worth mentioning `gp top` and `gp timeout extend`. Workspace stop, ports and tasks are likley the most useful commands, and the user can discover the rest if they need?